src/Service/MicrosoftGraphBookingService.php

```diff
@@ -456,31 +465,51 @@ public function getUserBookingFromApiData(array $data): UserBooking
             $userBooking->displayName = $data['location']['displayName'];
             $userBooking->body = $data['body']['content'];
 
-            $locationUniqueId = $data['location']['uniqueId'];
             $organizerEmail = $data['organizer']['emailAddress']['address'] ?? null;
 
             $userBooking->ownedByServiceAccount = $organizerEmail && mb_strtolower($organizerEmail) == mb_strtolower($this->serviceAccountUsername);
 
             $bookingType = $userBooking->ownedByServiceAccount ? UserBookingTypeEnum::ACCEPTANCE : UserBookingTypeEnum::INSTANT;
             $userBooking->bookingType = $bookingType->name;
 
-            // Find resource mail.
-            $attendeeResource = null;
+            // Extract the resource from the body of the event.
+            // Assumption: event body contains td with id = resourceMail.
+            $body = $data['body']['content'];
+            $doc = new DOMDocument();
+            $doc->loadHTML($body);
+            $xpath = new DOMXpath($doc);
+
+            $resourceMail = null;
+            $resourceName = null;
+
+            $resourceMailDOMNodes = $xpath->query("//td[@id='resourceMail']");
+            if (isset($resourceMailDOMNodes[0])) {
+                $resourceMail = $resourceMailDOMNodes[0]->textContent;
```

Comment:
It may be worth trimming the text content and using `!empty(…)` when checking for a value later.